packages/terra-site/src/examples/form-input/uncontrolled_examples/DefaultExample.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import FormInput from 'terra-form-input/lib/Input';
```

Comment:
Switch to `import Input from 'terra-form-input';` after renaming component in `packages/terra-form-input/src/Input.jsx`